draft-ietf-tls-tls13.md

```diff
@@ -2633,7 +2650,9 @@ filters
   some of the required certificate extension OIDs and supplied a certificate
   that does not satisfy the request, the server MAY at its discretion either
   continue the connection without client authentication, or abort the handshake
-  with an "unsupported_certificate" alert.
+  with an "unsupported_certificate" alert.  The client MAY abort the handshake
+  if a given certificate_extension_oid value appears multiple times in the
+  filters list.
```

Comment:
This is a new restriction, but I think it matches the intent.

Reply:
Seems weak.  Is there a prohibition on repeated values? Should there be?  If so, then this is an alertable offense.  If there are reasons to allow duplicate values, then we don't need this.

Reply:
I don't have a great understanding of how the filters actually work, myself, so I can only say that I *think* there is not a reason to allow duplicate values.  I also had in my head that this was going to be MUST when I first noticed it, so I'll go ahead and make the MAY-->MUST change for now -- thanks for pointing it out.

Reply:
So, it does seem like there might be a valid reason to provide multiple copies, or at least a coherent interpretation.

Specifically, because the semantics are all values must be present.

```
X = [A, B]
```
and
```
X=[A]
X=[B]
```

Seem equivalent.

I agree we have to resolve this, but I think MAY is the worst way, because then it creates interop problems. Either duplicates may appear or they may not, so we should say *that*